prow/cmd/peribolos/main_test.go

```diff
@@ -1035,9 +1047,29 @@ func TestConfigureTeam(t *testing.T) {
 				Privacy: string(secret),
 			},
 		},
+		{
+			name:     "patch team when parent changes",
+			teamName: whatev,
+			parent:   &parent,
+			config:   org.Team{},
+			github: github.Team{
+				ID:   3,
+				Name: whatev,
```

Comment:
Please make sure the parent ID is the only bad value.

By making `config.Name != github.Name` also differ it is no longer clear that `ParentTeamID` change causes an update (might just be luck due to the name diff).